src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx

```diff
@@ -88,21 +88,9 @@ It's also possible to review the state using the Nerdgraph API ('status' propert
   Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts & AI > Destinations**. Click the **Destinations** tab to see the destinations table.
 </figcaption>
 
-### Notifications log [#notifications-log]
+### Issue Notifications log [#notifications-log]
 
-To view past notification events details, go to the **Destination** menu, and click the **Notifications log** tab.
-
-A notifications log enables you to view the history and status of all your past notifications. Here you can view the status of any notification along with related error details and destination ticket numbers.
-
-<img
-  title="The filter tool that makes it easier to find the information you're looking for."
-  alt="A screenshot of the filter tool that makes it easier to find the information you're looking for."
-  src={accountsFilterDestinations}
-/>
-
-<figcaption>
-  Filter your destination logs by destination type, sent by, and status.
-</figcaption>
+The [Issue notification log](docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/#issue-notification-log) gives you a detailed record to keep track of all notifications generated by workflows. This information helps you troubleshoots issues, monitor workflow performance, and ensures that notifications are being sent to the correct destinations. As a result, you can quickly identify any issues that may arise and take corrective action as needed.
```

Comment:
@Matan2801 the link for "Issue notification log" seems to be broken. Could you double check it?

Reply:
@rhetoric101 and @Matan2801, fixed! The URL must start with the "/"

Reply:
@nbaenam thanks for your help with this!